internal/verifier/change_stream.go

```diff
@@ -52,8 +53,81 @@ func (uee UnknownEventError) Error() string {
 	return fmt.Sprintf("received event with unknown optype: %+v", uee.Event)
 }
 
+type ChangeStreamReader struct {
+	readerType clusterType
+
+	lastChangeEventTime *primitive.Timestamp
+	logger              *logger.Logger
+	namespaces          []string
+	metaDBName          string
+
+	metaClient    *mongo.Client
+	watcherClient *mongo.Client
+	buildInfo     *util.BuildInfo
```

Comment:
I don't think they will. They're pointers just to be consistent with the `verifier` struct.

Reply:
The pointer in `verifier` is a bit of a mystery, but I think it’s there because when that struct is initialized the BuildInfo isn’t loaded yet. So it’s “optional” in that way.

That’s not the case here, though: BuildInfo is _always_ available here. While consistency is a good thing on its own, maybe there’s more benefit in avoiding the notion that BuildInfo is “optional” here?